arcaflow_plugin_sysbench/sysbench_plugin.py

```diff
@@ -201,6 +201,7 @@ def RunSysbenchMemory(
 
     try:
         output, results = run_sysbench(memory_flags, "memory")
+        output["memory_access_mode"] = params.memory_access_mode
```

Comment:
This seems odd ... why are you reporting **only** the input memory address mode? Is this somehow particularly important compared to the other configuration params?

Reply:
Nearly all of the other input parameters are already reported in the normal output from `sysbench`, so we do not need to explicitly add them. The only other exception to that, I think, is the `memory_hugetlb` parameter.